src/java/fr/paris/lutece/plugins/workflow/modules/extendcomment/business/CommentNotificationTaskConfigDAO.java

```diff
@@ -0,0 +1,141 @@
+    private static final String SQL_QUERY_DELETE = "DELETE FROM task_forms_extend_comment_notification_cf WHERE id_task = ? ";
+    private static final String SQL_QUERY_UPDATE = "UPDATE task_forms_extend_comment_notification_cf SET id_task = ?, message = ?, subject = ?, sender_name = ?, sender_email = ? WHERE id_task = ?";
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void insert( CommentNotificationTaskConfig config )
+    {
+        try ( DAOUtil daoUtil = new DAOUtil( SQL_QUERY_INSERT, WorkflowCommentPlugin.getPlugin( ) ) )
+        {
+            int nIndex = 0;
+            daoUtil.setInt( ++nIndex, config.getIdTask( ) );
+
+            daoUtil.setString( ++nIndex, config.getMessage( ) );
+            daoUtil.setString( ++nIndex, config.getSubject( ) );
+            daoUtil.setString( ++nIndex, config.getSenderName( ) );
+            daoUtil.setString( ++nIndex, config.getSenderEmail( ) );
+
+            daoUtil.executeUpdate( );
+            daoUtil.free( );
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void store( CommentNotificationTaskConfig config )
+    {
+        try ( DAOUtil daoUtil = new DAOUtil( SQL_QUERY_UPDATE, WorkflowCommentPlugin.getPlugin( ) ) )
+        {
+            int nIndex = 0;
+            daoUtil.setInt( ++nIndex, config.getIdTask( ) );
+            daoUtil.setString( ++nIndex, config.getMessage( ) );
+            daoUtil.setString( ++nIndex, config.getSubject( ) );
+            daoUtil.setString( ++nIndex, config.getSenderName( ) );
+            daoUtil.setString( ++nIndex, config.getSenderEmail( ) );
+
+            daoUtil.setInt( ++nIndex, config.getIdTask( ) );
+
+            daoUtil.executeUpdate( );
+            daoUtil.free( );
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public CommentNotificationTaskConfig load( int nIdTask )
+    {
+        try ( DAOUtil daoUtil = new DAOUtil( SQL_QUERY_SELECT, WorkflowCommentPlugin.getPlugin( ) ) )
+        {
+            daoUtil.setInt( 1, nIdTask );
+            daoUtil.executeQuery( );
+
+            CommentNotificationTaskConfig commentNotificationTaskConfig = null;
+
+            if ( daoUtil.next( ) )
+            {
+                commentNotificationTaskConfig = new CommentNotificationTaskConfig( );
+                int nIndex = 0;
+
+                commentNotificationTaskConfig.setIdTask( daoUtil.getInt( ++nIndex ) );
+                commentNotificationTaskConfig.setMessage( daoUtil.getString( ++nIndex ) );
+                commentNotificationTaskConfig.setSubject( daoUtil.getString( ++nIndex ) );
+                commentNotificationTaskConfig.setSenderName( daoUtil.getString( ++nIndex ) );
+                commentNotificationTaskConfig.setSenderEmail( daoUtil.getString( ++nIndex ) );
+
+            }
+
+            daoUtil.free( );
```

Comment:
Remove this statement (daoUtil.free( )), the call to close the connection is already made implicitly because the DAUtil class implements the AutoCloseable interface